src/lib/passageClient.ts

```diff
@@ -118,9 +118,6 @@ class Passage {
         invite_token: params.inviteToken,
         password: params.password,
       },
-      headers: {
-        ContentType: 'application/json',
-      },
```

Comment:
`Content-Type: application/json` is the default, if no headers are specified. If there are any headers specified, then it will get overriden